aries-site/src/pages/components/selectmultiple.mdx

```diff
@@ -0,0 +1,47 @@
+import { Example } from '../../layouts';
+import { SelectMultipleAnatomyExample, SelectMultipleSimpleExample } from '../../examples';
+
+The SelectMultiple component is typically used as a form element but can also be leveraged for user experiences that include searching or narrowing down data. It contains an optional search field. When within a form, the SelectMultiple will support labels, error and disabled states.
+
+<Example
+  componentName="SelectMultiple"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/selectmultiple/SelectMultipleSimpleExample.js"
+  docs="https://v2.grommet.io/selectmultiple?theme=hpe#props"
+  grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/SelectMultiple/SelectMultiple.js"
```

Comment:
add figma link

Reply:
Couldn't find it. Do you have the link or do we need to create an official one?

Reply:
Im guessing we will use this one once everything is moved into finished 
https://www.figma.com/file/EcLSFz5L7EwmOadmEYGshZ/SelectMultiple-Component?node-id=1%3A1026